packages/stateless/src/lib/provider/provider.interface.ts

```diff
@@ -28,7 +28,7 @@ export interface IEmailOptions {
 
 export interface ISmsOptions {
   to: string;
-  content: string;
+  content: any;
```

Comment:
I disagree with that change - content should be always a string and is expected to be a string by other providers.